### PR TITLE
Sensitive Data API and order physical card api integrated

### PR DESCRIPTION
**Pull request by hiteshdigit101**
View PAN/CVV and order physical card API integrated.

### CODE REVIEW

File: src/components/adb-card/card-info-component/components/virtual-card/index.tsx [hunk 1]
```diff
@@ -6,18 +6,19 @@ import {
   TouchableOpacity,
   View,
 } from "react-native";
-import { ThemeContext } from "react-native-theme-component";
+import { LoadingModal, ThemeContext } from "react-native-theme-component";
 import { CloseEyesIcon } from "../../../../../assets/close-eyes.icon";
 import { CopyIcon } from "../../../../../assets/copy.icon";
 import { EyesIcon } from "../../../../../assets/eyes.icon";
 import { LockIcon } from "../../../../../assets/lock-icon";
 import VisaIcon from "../common/visa-icon";
```
Comment (sharmal33) [inline src/components/adb-card/card-info-component/components/virtual-card/index.tsx:14]: can you cleanup this
eg :
import { CopyIcon,CloseEyesIconEyesIcon,... } from "../../../../../assets/icon";

File: src/components/adb-card/card-info-component/components/virtual-card/index.tsx [hunk 2]
```diff
@@ -167,43 +182,52 @@ const VirtualCard: React.FC<VirtualCardProps> = (props) => {
               <View style={styles.cardDetail}>
                 <Text style={styles.visaCardText}>CVV</Text>
```
Comment (sharmal33) [inline src/components/adb-card/card-info-component/components/virtual-card/index.tsx:183]: move this into translations file

File: src/components/adb-card/card-info-component/components/virtual-card/index.tsx [hunk 3]
```diff
@@ -167,43 +182,52 @@ const VirtualCard: React.FC<VirtualCardProps> = (props) => {
...
-            </View>
-            <View style={styles.marginH}>
-              <Text style={styles.cardNumberText}>••••</Text>
-            </View>
-            <View style={styles.marginH}>
-              <Text style={styles.cardNumberText}>
-                {showCardNumber ? cardNumber.substring(12, 16) : "••••"}
-              </Text>
-            </View>
+            <View style={{ flexDirection: "row", width: "80%" }}>
```
Comment (sharmal33) [inline src/components/adb-card/card-info-component/components/virtual-card/index.tsx:201]: inline style

File: src/components/adb-card/card-info-component/components/virtual-card/index.tsx [hunk 4]
```diff
@@ -240,54 +263,69 @@ const VirtualCard: React.FC<VirtualCardProps> = (props) => {
...
-              }}
+              labelColor={colors.btnColor}
+              background={colors.white}
+              label={i18n?.t("adb_card.btn_go_home")}
+              onPress={() => setAlert(false)}
             />
+            <View
+              style={{
+                marginTop: 10,
+              }}
```
Comment (sharmal33) [inline src/components/adb-card/card-info-component/components/virtual-card/index.tsx:295]: inline style

File: src/components/adb-card/physical-card-component/enter-app-passcode/index.tsx [hunk 5]
```diff
@@ -279,9 +265,7 @@ const EnterAppPassCodeComponent: React.FC<AppPassCodeProps> = (
       >
         <BackIconArrow />
       </TouchableOpacity>
-      <View
-        style={{ paddingBottom: 40 }}
-      >
+      <View style={{ paddingBottom: 40 }}>
```
Comment (sharmal33) [inline src/components/adb-card/physical-card-component/enter-app-passcode/index.tsx:268]: inline style

File: src/context/wallet-context.ts [hunk 6]
```diff
@@ -1714,6 +1730,35 @@ export function useWalletContextValue(): WalletContextData {
     [_cardWallet]
   );
 
+  const getSensitiveData = useCallback(async () => {
+    setIsShowLoadingSensitiveData(true);
+    setSensitiveData(undefined);
+    try {
+      const publicKey = `MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEAzHgyZFwqjK8rECfSJB2cvFh/vV/x+svcrIhv8rT47fzvZKATisKA9D74k1s4h7RV6nUPIzNnquN7dNFKxiWxbkOMzPDLOfuzySXPsGRdAq7D6RhOnyAioMkU6eSk9oUX+VApNyHCBe1fHU3RPwkoHK4ktPieR1FR0C+sO/fT7+nyNKh8mwC986jJ7SZnJoPFegvTrdmeIv6o7fNJ0+qru7KE6QyDdl0EikRvHGq/DsfMGpLSHwUzq8Wl6mjs4QsClmNB+R+qzlODnfdf6RZAJXgai0Yr0YNBy7eJN+iCtQNsnooz9lBtIqTggyUb3quWTmfAhg5cbIRLlBik83pcA7M0hoMBPSuefFHEkic7G67B8/ikcCX7jQYU/iiKiSzXtWEQwjwk65jaNR/6h7/02H5Ja3N43o3Fcm7U5Uo3nbcQqYcNF0XKUkAz0LI2BxOX1GPNPVOHZdYNZKm7Nr+RHyHvUH+WaeCO1cdKVAAlO1rzUs+KFJGxvt7IIMHsZk69XwsVIA35r4znhb6xf36PJK/cwrRi+PszkxipFozVIzeADgjC5Mgu9tYpyd+cLFy57tQHgz4PkcCFW7H3fxWKwdaQb3DB/xjVsiiJ6mwflqQq0qx6Re2m7i0mSnQgXVBrLuOA4AZiskN+aliqGw+DQGjq/I7OWPOeykjE2ZphxP8CAwEAAQ==`;
```
Comment (sharmal33) [inline src/context/wallet-context.ts:1737]: move this into env

**Review by sharmal33 — CHANGES_REQUESTED** [review]

File: src/components/adb-card/card-info-component/components/virtual-card/index.tsx (hunk 1 above)
Comment (hiteshdigit101) [reply src/components/adb-card/card-info-component/components/virtual-card/index.tsx]: Done

File: src/components/adb-card/card-info-component/components/virtual-card/index.tsx (hunk 2 above)
Comment (hiteshdigit101) [reply src/components/adb-card/card-info-component/components/virtual-card/index.tsx]: Moved to translation file

File: src/components/adb-card/card-info-component/components/virtual-card/index.tsx (hunk 3 above)
Comment (hiteshdigit101) [reply src/components/adb-card/card-info-component/components/virtual-card/index.tsx:214]: Removed Inline styles

File: src/components/adb-card/card-info-component/components/virtual-card/index.tsx (hunk 4 above)
Comment (hiteshdigit101) [reply src/components/adb-card/card-info-component/components/virtual-card/index.tsx]: Removed Inline styles

File: src/components/adb-card/physical-card-component/enter-app-passcode/index.tsx (hunk 5 above)
Comment (hiteshdigit101) [reply src/components/adb-card/physical-card-component/enter-app-passcode/index.tsx]: Removed Inline styles

File: src/context/wallet-context.ts (hunk 6 above)
Comment (hiteshdigit101) [reply src/context/wallet-context.ts]: moved to env file

**Pull request merged**